### ISSUE TITLE
Redirect loop when login into manager

### ISSUE THREAD

**Issue by benjaminbascary** [ragapp/ragapp#166]
**Describe the bug**
After a first successful login into the manager, the next time one tries to login, the app gets stuck in a redirect loop.

**To Reproduce**
Login into manager
Login into keycloak
Logout keycloak
Try to login again in the manager with no success
Changing to another device with same behavior
Changing browser with same behavior

**Expected behavior**
Login into manager successfully every time, not just the first time

**Screenshots**
EC2 docker compose logs from the very start when I run `docker-compose up`:
https://gist.github.com/benjaminbascary/965c6650509949270be7055c02d0869c

Video:

https://github.com/user-attachments/assets/81bbf571-f9b7-4092-a260-c05f0179ffc6

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.5
 - Browser: safari and arc

**Additional context**
Tried clearing cookies and also the session from the keycloak admin, with no luck
Pruning all containers does not work also

Deployed this into intelligence.patagonian.com (if you want to try, there is nothing confidential so go ahead and login with the default credentials)

**Comment by leehuwuj** [comment]
Hi @benjaminbascary , Sorry i could not reproduce you issue with the default deployment settings. But from you provided log, it is said that the Traefik is not allowed to connect with Keycloak, see:
```
keycloak-1  | 2024-09-05 14:59:11,867 WARN  [org.keycloak.events] (executor-thread-3) type="INTROSPECT_TOKEN_ERROR", realmId="39f6a9d1-3807-4e74-943e-c62445b3b618", realmName="ragapp", clientId="ragapp", userId="null", ipAddress="172.23.0.2", error="invalid_request", detail="Client not allowed.", client_auth_method="client-secret"
```
If you had changed the default realm credentials, please make sure you updated it for Keycloak config too.

Run `docker compose down` and run `docker compose up` again does not fix the issue because the update on Keycloak is persisted on dist (`$STATE_DIR/keycloak` folder)

**Comment by benjaminbascary** [comment]
Hi @leehuwuj , sorry the late response. I don't know what I changed to break my authentication. Not the credentials, that is for sure.

I was tweaking some things in the Keycloak dashboard related to the manager realm. Im sure I changed something and I did not notice. You were right, I reverted the changes on the keycloak db file, re ran the docker services and all got up again. 

Thank you! and sorry for the inconvenience, it was on my side.
